frontend/src/actions/cards.js

```diff
@@ -40,6 +41,21 @@ export function fetchCardDetail(cardId) {
   };
 }
 
+export function updateUserCards(cardId) {
+  return dispatch => {
+    return apiFetch('cards/' + cardId + '/transaction', {
+      method: 'PUT',
+      body: JSON.stringify(cardId)
+    })
+      .then(response => response.json())
+      .then(json => {
+        if (json.success) {
+          toast.success('Card successfuly purchased!');
```

Comment:
successfuly -> successfully

Reply:
lol, i think i typed that line on lena's computer 😬